### PR TITLE
Fact measurements must be mandatory, while statements and original are optional

### PR DESCRIPTION
**Pull request by fiunchinho**
#### Submitter checklist

- [X] Change is code complete and matches issue description.
- [X] Change is covered by existing or new tests.

#### Description
Correctly set the required and optional fields for the Fact CRD.

### CODE REVIEW

File: pkg/apis/jenkins.io/v1/types_fact.go [hunk 1]
```diff
@@ -39,10 +39,12 @@ type Fact struct {
 type FactSpec struct {
 	Name     string `json:"name" protobuf:"bytes,1,opt,name=name"`
```
Comment (jglick) [inline pkg/apis/jenkins.io/v1/types_fact.go]: What is this?

**Comment by jglick** [comment]
amends #3066 FTR

**Comment by jstrachan** [comment]
@fiunchinho there is a formatting error in the PR. I wonder could you run `make fmt` and push again

**Comment by fiunchinho** [comment]
It's fixed now!

**Pull request merged**